test/decompression.jl

```
@@ -0,0 +1,25 @@
+module TestCompression
+using Test
+using DataFrames
+
+using RData
+
+rdata_path = joinpath(dirname(@__FILE__), "data_v3")
+df = DataFrame(num = [1.1, 2.2])
+
+@testset "When the optional compression codecs are not loaded" begin
+    @test_throws ErrorException load(joinpath(rdata_path, "compressed_bzip2.rda"))
```

We should probably throw a custom exception or check the error message explicitly. I need to work out how to do that.

---

+1 for custom exception (`RDataException`). There's #14, which would be nice to implement at some point. But it deserves a separate PR anyway, I think this test is fine for now.

---

@alyst I've added a custom exception. Let me know if this is ok.

I decided not to apply it more broadly as I felt that would be off-topic for this PR.